src/react/account/details/Locations.jsx

```diff
@@ -122,7 +122,7 @@ function Locations() {
     }, useFilters, useSortBy);
 
     // NOTE: empty state component
-    if (locations.length === 0) {
+    if (Object.keys(locations).length === 0) {
```

Comment:
Why do we need to use `Object.keys` method?

Reply:
Because `locations` is an object and not an array. (typeof Locations)
```
export type Locations = $ReadOnly<PerLocationMap<Location>>;
export type PerLocationMap<T> = {
    [LocationName]: T,
};
```

```
console.log([].length) // 0
console.log({}.length) // undefined
```

Reply:
Good catch, thank you :)